rdtools/normalization.py

```diff
@@ -271,11 +277,22 @@ def normalize_with_sapm(energy, sapm_kws):
 
 
 def delta_index(series):
-    # Takes a panda series as input and returns (time step sizes, average time step size)
-    # Length of each interval calculated by using 'int64' to convert to nanoseconds
+    '''
+    Takes a panda series as input and returns (time step sizes, average time step size) in hours
```

Comment:
Is it worth noting here that the pandas series index must be a DatetimeIndex?

Reply:
I think that it is, I have added that.